docs/connectors.md

```diff
@@ -0,0 +1,40 @@
+## Connector Maintainers
+
+* **Official** -  Official connectors and tools are built, maintained, supported, and tested by Meltano.
+* **Partner** - Partner connectors and tools are built, maintained, supported, and tested by a partner organization of Meltano, including consultancies and other vendors.
+* **Community** - Community connectors and tools are built and maintained by the wider open source community.
+
+## Connector Quality
+
+* **Gold** - Gold connectors are built on the Meltano SDK, are actively used by many Meltano Cloud and open source users, and have broad coverage of data and features.
+* **Silver** - Silver connectors may be built on the Meltano SDK, but their open source and Meltano Cloud usage may be lower compared to a Gold connector.
+* **Bronze** - Bronze connectors are not based on the Meltano SDK. We have some usage data for this connector in Meltano Core or Cloud.
+* **No Data** - Connectors for which we have no usage data. They likely have not been tested and the repository maintainers may be unresponsive.
+
+
+### Connector Quality Matrix
+
+To give users a better understanding of the overall quality of a connector, we have the following matrix and guidance on attributes that affect the quality rating.
+
+|         | SDK-Based | Usage Data    | Maintainer         | Repo Activity  |
+|---------|-----------|---------------|--------------------|----------------|
+| Gold    | Yes       | > 5 Projects  | Meltano or Partner | High           |
```

Comment:
@tayloramurphy Wouldn't a _community_-maintained connector "built on the Meltano SDK, are actively used by many Meltano Cloud and open source users, and have broad coverage of data and features" also be Gold? 

```suggestion
| Gold    | Yes       | > 5 Projects  | Official, Partner, or Community | High           |
```

Reply:
@DouweM I think that's fair. When I set this up originally I was thinking Gold would be a bonus for Partner-maintained connectors, but it's not necessary at this stage.